packages/juce-blueprint/esm/lib/BlueprintBackend.js

```diff
@@ -24,12 +24,73 @@ if (typeof window !== 'undefined') {
   };
 }
 
+function isMouseEventHandler(key) {
+  const k = key.toLowerCase();
+
+  return k === 'onmousedown' ||
+    k == 'onmouseup' ||
+    k == 'onmousedrag' ||
+    k == 'onmouseclick';
+}
+
+function isKeyboardEventHandler(key) {
+  const k = key.toLowerCase();
+
+  return k === 'onkeydown' ||
+    k == 'onkeyup' ||
+    k == 'onkeypress';
+}
+
+class SyntheticEvent {
+  constructor(props) {
+    this.bubbles = true;
```

Comment:
No idea whether it's worth making this cancelBubble for parity? https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelBubble

Reply:
Yea, IIRC `cancelBubble` is deprecated, and `bubbles` should be a read-only property. I'm tempted to say we can ignore adding deprecated features?

Reply:
Oh, jk, it is not deprecated! It's just an alias to `stopPropagation()`. Ok, yea, should update